.github/workflows/build-workflow.yml

```diff
@@ -37,20 +37,32 @@ jobs:
           name: debian-packages-amd64
           path: target/debian/*.deb
 
+      - name: Publish debian packages (for internal use)
```

Comment:
Now the workflow name has to be `build-and-publish.yml`

Reply:
Yes that is probably worth while to do to make it more transparent. I originally decided against it as it was only publishing internally, however it makes it easier to find if it includes the publishing in the name.

Reply:
After reviewing the renaming together we decided to change the job name, not the workflow name.

Work is planned to refine/optimize the build/ci/pr-check/release process where the workflow naming will be revisited.